modules/cluster-logging-updating-logging-to-5-1.adoc

```diff
@@ -0,0 +1,214 @@
+[id="cluster-logging-updating-logging-to-5-1_{context}"]
+= Updating OpenShift Logging to the current version
+
+To update OpenShift Logging from 5.x to the current version, you change the subscriptions for the OpenShift Elasticsearch Operator and Red Hat OpenShift Logging Operator.
+
+You must update the OpenShift Elasticsearch Operator _before_ you update the Red Hat OpenShift Logging Operator. You must also update _both_ Operators to the same version.
```

Comment:
Again, not sure if this deserves being emphasized with an admonition?  (Note, Caution)

Reply:
Updated. Thanks.